app/bundles/WebhookBundle/Form/Type/ConfigType.php

```diff
@@ -64,7 +65,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * {@inheritdoc}
      */
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
You have to remove the `alias` from the config.php. Also, search for this alias in all Mautic files and replace it with the form class name. That must be done for all form types in the bundle.